src/components/AddComment.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import { postComment } from "../api";
+
+const AddComment = ({comments, review_id}) => {
+    const [commentAuthor, setCommentAuthor] = useState('');
+    const [commentBody, setCommentBody] = useState('');
+    const [commentVotes, setCommentVotes] = useState(0);
+    const [commentPosted, setCommentPosted] = useState(false)
+
+    return <section>
+        <form className="add-comment-form" onSubmit={(event) => {
+            event.preventDefault()
+
+            const commentToAdd = {
+                "author": commentAuthor,
+                "body": commentBody,
+                "votes": commentVotes
+            };
+            setCommentPosted(true)
+            postComment(review_id, commentToAdd)
+            .then((newComment) => {
+                comments((currentComments) => {
+                    return [newComment, ...currentComments];
+                })
+            })
+            .then(() => {
+                setCommentAuthor('');
+                setCommentBody('');
+                setCommentVotes(0);
```

Comment:
We will also want to add a catch block here to conditionally display an error message to the user if a post submission has not gone through successfully. Think about managing this using some error state